frame/contracts/src/tests.rs

```diff
@@ -108,10 +108,11 @@ pub mod test_utils {
 	}
 }
 
-thread_local! {
-	static TEST_EXTENSION: RefCell<TestExtension> = Default::default();
+parameter_types! {
+	static TestExtensionsTestValue: TestExtension = Default::default();
```

Comment:
Why is this renamed?

Reply:
Still a valid question?

Reply:
There's a struct named `TestExtension`, would cause a conflict.